src/python_routing_v02/fast_reach/mc_reach.pyx

```diff
@@ -191,30 +191,15 @@ cpdef object compute_network(
         # # FlowVelDepth[fill_index]['flow'] = UpstreamOutflows[upstream_tw_id]['flow']
         # # FlowVelDepth[fill_index]['depth'] = UpstreamOutflows[upstream_tw_id]['depth']
 
-    # for ts in flowveldepth:
-        # print(f"{list(ts)}")
-
-    cdef set fill_index_mask = set()
-    # fill_index_mask is filled in the explicit loop below, which is
-    # identical to the following comprehension. But we use the explicit loop, because it
-    # is more transparent (and therefore optimizable) to cython.
-    # cdef set fill_index_mask = set([upstream_results[upstream_tw_id]["position_index"] for upstream_tw_id in upstream_results])
-    #print(f"{fill_index_mask}")
+    fill_index_mask = np.ones_like(data_idx, dtype=bool)
+    cdef dict tmp
```

Comment:
not sure if cdef is needed here or not. With gil at this point and these variables are not used until after the gil is back at the end of the script